lib/utils/messages.ts

```diff
@@ -44,24 +44,16 @@ const getMessage = function (messageType: string, ...args: any[]) {
       return `No matching message prefix: ${args[0]}`;
     case 'METRIC_IS_UNAVAILABLE':
       return `Sorry, ${args[0]} metric is unavailable`;
-    case 'ttfcp':
+    case 'first-meaningful-paint':
```

Comment:
seems wrong.

in 3.0 FCP is both in 'metrics' audit and its own. i would def recommend showing **both** FCP and FMP in pwmetrics. (plots doesnt have FCP)